greeter/README.md

```diff
@@ -302,7 +307,7 @@ Let's try it one more time with language `fr`. Since there is no greeter for Fre
 
 ## Part 2: Add a French Greeter
 
-As we mentioned above, LB4 allows you to extend your application by binding it with other modules/extensions to meet your business requirements. The following steps show how LB4 uses extension point to enrich the functionality of your greeting app.
+LB4 allows you to extend your application by binding it with other modules/packages to meet your business requirements. LB4 implements the design pattern extension points/extensions, which organize artifacts with loose coupling and promote extensibility. We have showed in previous steps that our app has two packages. one of the component in `greeting-extension` is `GreetingComponent`. `GreetingComponent` already has extension points defined and has two greeters plugged in. The following steps show how to uses the extension point in `GreetingComponent` to plug in an extension to enrich the functionality of your greeting app.
```

Comment:
> LB4 allows you to extend your application by binding it with other modules/packages to meet your business requirements. LB4 implements the design pattern extension points/extensions, which organize artifacts with loose coupling and promote extensibility. We have showed in previous steps that our app has two packages. one of the component in `greeting-extension` is `GreetingComponent`. `GreetingComponent` already has extension points defined and has two greeters plugged in. The following steps show how to uses the extension point in `GreetingComponent` to plug in an extension to enrich the functionality of your greeting app.


LB4 allows you to extend your application by importing other modules and binding/injecting various artifacts to meet your business requirements. LB4 implements the [extension point/extensions](https://loopback.io/doc/en/lb4/Extension-point-and-extensions.html) design pattern , which organizes artifacts with loose coupling and promotes extensibility. We have shown in previous steps that our app is organized into two modules. The `greeting-extension` module defines a `GreetingComponent`. `GreetingComponent` contains `GreetingService` which declares an extension point named `greeters`.  There are 2 greeters (EnglishGreeter and ChineseGreeter which implement the Greeter interface, and which register themselves as extensions to this extension point `greeters`. `GreetingService` looks for extensions that can handle a particular language. If it cannot find the greeter for a particular language, it defaults to English. The following steps show how to use the extension point `greeters` in `GreetingComponent` to plug in an extension (FrenchGreeter) to enrich the functionality of your greeting app.